### PR TITLE
Delete firmware files

### PR DESCRIPTION
**Pull request by bzapa**

### CODE REVIEW

File: app/src/main/java/programatorus/client/comm/transport/io/StreamingTransport.kt [hunk 1]
```diff
@@ -191,6 +193,10 @@ abstract class StreamingTransport<T : StreamingTransport<T>>(
         }
 
         state = ConnectionState.DISCONNECTED
+
+        val intent = Intent("programatorus,client.DISCONNECT")
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/comm/transport/io/StreamingTransport.kt]:
```suggestion
        val intent = Intent("programatorus.client.DISCONNECT")
```

File: app/src/main/java/programatorus/client/screens/home/HomeFragment.kt [hunk 2]
```diff
@@ -45,6 +50,10 @@ class HomeFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        showLoading()
+
+        requireContext().registerReceiver(disconnectHandler, IntentFilter("programatorus,client.DISCONNECT"))
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/screens/home/HomeFragment.kt]:
```suggestion
        requireContext().registerReceiver(disconnectHandler, IntentFilter("programatorus.client.DISCONNECT"))
```

File: app/src/main/proto/protocol.proto [hunk 3]
```diff
@@ -188,6 +192,8 @@ message GenericMessage {
     DebuggerLine debuggerLine = 207;
     // Response: Ok(102)
 
+    DeleteFile deleteFile = 208;
```
Comment (bgrzesik) [inline app/src/main/proto/protocol.proto:195]:
```suggestion
    DeleteFile deleteFile = 208;
    // Response: Ok(102)
```

**Review by bgrzesik — DISMISSED** [review]

File: app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt [hunk 4]
```diff
@@ -34,16 +51,41 @@ class DebuggerFragment : Fragment() {
...
+        future = CompletableFuture()
+        loop()
+    }
+
+    private fun loop() {
+        device.pollDebuggerLine().thenAccept { line ->
+            Log.d(TAG, "line polled: $line")
+            receive(line)
+            loop()
+        }
```
Comment (bzapa) [inline app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt]: future never completes

Comment (bgrzesik) [reply app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt]: Fixed in https://github.com/bgrzesik/programatorus-server/pull/18

**Review by bgrzesik — APPROVED** [review]

**Pull request merged**